### DISCUSSION TITLE
process in App.vue Uncaught ReferenceError: process is not defined

### DISCUSSION THREAD

**Discussion by X-jc** [web-infra-dev/rsbuild#3680] [Q&A] [answered]
在App.vue文件中获取process.env.BASE_URL报错Uncaught ReferenceError: process is not defined。
![test](https://github.com/user-attachments/assets/9658db51-5061-4289-bb0e-221f49e68449)

process或者process.env.BASE_URL应该是在全局中都可以获取到的吧？还是我需要在define中定义他？
const { publicVars } = loadEnv({ prefixes: ['VUE_APP_'] });
  source: {
    // 指定入口文件
    entry: {
      index: './src/main.ts',
    },
    define: {
      ...publicVars
    },
  },
因为我的项目是从vue-cli迁移，目前我做了上述的处理。

**Comment by 9aoy** [comment, marked as answer]
`process.env.BASE_URL` is supported in rsbuild `v1.0.10`. You can try upgrading the version.
https://rsbuild.dev/config/server/base
